packages/flutter/test/material/card_test.dart

```diff
@@ -39,22 +39,25 @@ void main() {
         children: <TestSemantics>[
           new TestSemantics.rootChild(
             id: 1,
-            label: 'I am text!\nMoar text!!1',
+            label: 'I am text!',
             textDirection: TextDirection.ltr,
-            children: <TestSemantics>[
-              new TestSemantics(
-                id: 2,
-                label: 'Button',
-                textDirection: TextDirection.ltr,
-                actions: <SemanticsAction>[
-                  SemanticsAction.tap,
-                ],
-                flags: <SemanticsFlag>[
-                  SemanticsFlag.isButton,
-                  SemanticsFlag.hasEnabledState,
-                  SemanticsFlag.isEnabled,
-                ],
-              ),
+          ),
+          new TestSemantics.rootChild(
+            id: 2,
+            label: 'Moar text!!1',
+             textDirection: TextDirection.ltr,
+          ),
+          new TestSemantics.rootChild(
+            id: 3,
+            label: 'Button',
+            textDirection: TextDirection.ltr,
+            actions: <SemanticsAction>[
+              SemanticsAction.tap,
+            ],
+            flags: <SemanticsFlag>[
+              SemanticsFlag.isButton,
+              SemanticsFlag.hasEnabledState,
+              SemanticsFlag.isEnabled,
```

Comment:
Can you add a test that verifies that if the card just contains text, it still ends up being only a single node?

Reply:
Hm, I guess this won't really work. I'm not sure we have a way to express this in our current system.